aimon/extensions/react.py

```diff
@@ -0,0 +1,168 @@
+from aimon import Client
+from typing import Optional
+from dataclasses import dataclass
+
+@dataclass
+class ReactConfig:
+    publish: bool
+    max_attempts: int
+    aimon_api_key: str
```

Comment:
lets make this optional. Add a default 
`aimon_api_key:str = os.getenv("AIMON_API_KEY")`